syntax/crisp_lexer.mll

```diff
@@ -191,3 +229,12 @@ rule main = parse
   | "|" {BAR}
 
 (*FIXME string primitives as keywords -- e.g., concat, etc*)
+
+{
+let at_start_of_line lexbuf =
+  lexbuf.lex_curr_pos = 0
+  || (lexbuf.lex_curr_pos = lexbuf.lex_curr_p.pos_bol)
```

Comment:
I didn't understand this disjunct, and was wondering what value is encoded by `lexbuf.lex_curr_p.pos_bol` -- it would be useful to have a short comment adjacent to the code.